src/App.js

```diff
@@ -1,14 +1,23 @@
 import React, { Component } from 'react';
 import { data } from './data';
-import { ShoppingList } from './components';
+import { Wishlist, Cart } from './components';
 class App extends Component {
-  state = { list: data };
+  state = { list: data, newList: [] };
```

Comment:
What is this `newList` state doing here! We removed an unused state to add another one??!

Reply:
Oh